goreleaser.dockerfile

```diff
@@ -1,2 +1,3 @@
 FROM --platform=linux/amd64 alpine:3.20
 COPY transfer /transfer
+ENTRYPOINT ["/transfer"]
```

Comment:
How's this going to work if we need to pass in a config file via the CLI?

Reply:
The commands you enter in the CLI are appended to the `ENTRYPOINT`.